__tests__/test-data.test.js

```diff
@@ -50,3 +49,54 @@ describe("api/topics", () => {
     );
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns article api ", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200,check if it returns the right keys ", () => {
+    return request(app)
+      .get("/api/articles")
```

Comment:
This still isn't the right endpoint, we are looking for one single article, not all of them.